app/resources/add_note_base.html.template

```diff
@@ -15,12 +15,18 @@
 {% if not markdup %}
   <input type="hidden" name="add_note" value="yes">
   <input type="hidden" name="id" value="{{params.id}}">
-{% endif %} 
+{% endif %}
 
 <div class="status section">
   {% if not markdup %}
     <h2>Status</h2>
-    <div class="status subsection" id="status_info">
+    <div class="status subsection" id="status_info"
+      {% if env.enable_javascript %}
+         {% if params.create_mode %}
+            style="display: none"
```

Comment:
This is prone to inconsistencies (you have to make sure that this is the same as the updateForm logic).
Can you call updateForm as soon as the form loads instead?